packages/console/src/pages/Users/index.tsx

```diff
@@ -42,7 +42,7 @@ const Users = () => {
   const keyword = query.get('search') ?? '';
   const { data, error, mutate } = useSWR<[User[], number], RequestError>(
     `/api/users?page=${pageIndex}&page_size=${pageSize}&hideAdminUser=true${conditionalString(
-      keyword && `&search=%${keyword}%`
+      keyword && `&search=${encodeURIComponent(`%${keyword}%`)}`
```

Comment:
Just curious, why do we have `%` around the keyword?

Reply:
just like the `%` in sql, here is part of the `like`